### PR TITLE
[stable/fluentd-elasticsearch] fix ClusterRoleBinding apiGroups defaults

### PR DESCRIPTION
**Pull request by fabriziofortino**
#### What this PR does / why we need it:
This fixes the `apiGroups` defaults for `ClusterRoleBinding`. More details on why we need this here https://github.com/argoproj/argo-cd/issues/620

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

File: incubator/fluentd-elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,21 +0,0 @@
-name: fluentd-elasticsearch
```
Comment (davidkarlsen) [inline incubator/fluentd-elasticsearch/Chart.yaml]: was it intentional to delete the incubator-files?

Comment (fabriziofortino) [reply incubator/fluentd-elasticsearch/Chart.yaml]: yes, fluentd-elasticsearch is already in the stable folder. Should we maintain it in both folders?

Comment (davidkarlsen) [reply incubator/fluentd-elasticsearch/Chart.yaml]: See https://github.com/helm/charts/blob/master/PROCESSES.md for how that works

Comment (fabriziofortino) [reply incubator/fluentd-elasticsearch/Chart.yaml]: @davidkarlsen the process does not explain when a deprecated chart should be removed. I thought when a chart is promoted as stable the deprecated one could be removed. Anyway, I reverted my last commit. ty

**Pull request merged**